init.js

```diff
@@ -1,162 +1,106 @@
-'use strict';
+const events = require('events');
+const Config = require('./lib/config');
+const Hapi = require('hapi');
+const Registry = require('./lib/registry');
```

Comment:
@adslaton - fixed the bug noted below... but constants placed on individual lines is just a design decision I think is neater.